src/store/user/appSlice.ts

```diff
@@ -49,13 +53,15 @@ export const restoreUser = createAsyncThunk<Restore>(
   'app/restoreUser',
   async () => {
     const vals = await Promise.all([
-      getSecureItem('accountBackedUp'),
```

Comment:
you could potentially destructure this like `const [accountBackedUp, requirePin, ...] = await Promise.all([...])` if you don't want to keep track of which index each item is in

Reply:
yes! That's so much better.